public/canvas-customization/index.js

```diff
@@ -36,16 +36,16 @@
  * Load Blue Integration
  */
 var BLUE_CANVAS_SETUP={
-      connectorUrl:"https://surveys.berkeley.edu/berkeleySurveysblueconnector/",
-      canvasAPI:"https://bcourses.berkeley.edu",
-      domainName:"com.explorance",
-      consumerID:"CWN1RxHsa4vY9b2QgVC7jQ==",
-      defaultLanguage:"en-us"
-    },
+    connectorUrl:"https://surveys.berkeley.edu/berkeleySurveysblueconnector/",
+    canvasAPI:window.location.origin,
+    domainName:"com.explorance",
+    consumerID:"CWN1RxHsa4vY9b2QgVC7jQ==",
+    defaultLanguage:"en-us"
+},
     BlueCanvasJS=document.createElement("script");
 BlueCanvasJS.setAttribute("type","text/javascript");
 BlueCanvasJS.setAttribute("src","https://surveys.berkeley.edu/berkeleySurveysblueconnector//Scripts/Canvas/BlueCanvas.min.js");
-document.getElementsByTagName("head")[0].appendChild(BlueCanvasJS);
+BlueCanvasJS.async=!0;document.getElementsByTagName("head")[0].appendChild(BlueCanvasJS);
```

Comment:
@mlincoln - Would a newline after the semi-colon make this more readable?

Reply:
@mlincoln - Also, the expression `!0` will always be true. Is this syntax recommended?

Reply:
After line 47?

Reply:
I'm suggesting a line break after `BlueCanvasJS.async=!0;`

Reply:
Thanks, I added a line break after that semicolon, but I left the !0 piece as-is--I'm just copy/pasting code from Explorance.

Reply:
As a consequence of this PR I learned that `!0` and `!1` is standard Javascript minification for expressing `true` and `false` in the fewest possible characters. In general we don't bother with minifying our custom Canvas JS, but having it in there won't hurt anything, other than perhaps confusing future generations.